client/src/api/api.js

```diff
@@ -1,6 +1,6 @@
 const signUpCall = async (user) => {
   try {
-    const result = await fetch('/api/register', {
+    const result = await fetch('http://localhost:3001/api/register', {
```

Comment:
Can change this back to /api/register, I guess this was because of debugging yesterday

Reply:
we dont need to mention 3001 , proxy will do for us